src/grb_advroutines.jl

```diff
@@ -0,0 +1,53 @@
+mutable struct GRBsvec
+    len::Cint
+    ind::Ptr{Cint}
+    val::Ptr{Cdouble}
+end
+
+function get_tableaurow_grb(model::Model, cidx::Integer)
+    len = num_vars(model)+num_constrs(model)
+    idx = Array{Cint}(len)
+    val = Array{Cdouble}(len)
+
+    grb_v = GRBsvec(0, pointer(idx) , pointer(val))
+    ret = @grb_ccall(BinvRowi, Cint, (
+        Ptr{Void},
+        Cint,
+        Ref{GRBsvec}),
+        model, convert(Cint, cidx-1), grb_v)
+    if ret != 0
+        throw(GurobiError(model.env, ret))
+    end
+    return idx[1:grb_v.len]+1, val[1:grb_v.len]
+end
+
+function get_tableaurow!(row::Vector{Float64}, model::Model, cidx::Integer)
+    idx, val = get_tableaurow_grb(model, cidx)
+    row[idx] = val
+    return nothing
+end
```

Comment:
I'd prefer not to have these helper functions in Gurobi.jl. They don't get tested and lead to multiple ways to do the same thing.

Reply:
Do you prefer to remove get_tableaurow! ?